boards/nucleo-f401/include/periph_conf.h

```diff
@@ -103,6 +102,48 @@ static const uart_conf_t uart_config[] = {
 /** @} */
 
 /**
+ * @brief   PWM configuration
+ * @{
+ */
+static const pwm_conf_t pwm_config[] = {
+    { /* D3 */
+        .dev      = TIM2,
```

Comment:
I would prefer not to use TIM2 for PWM, as the same timer is already mapped to the `timer_` interface. If really wanted/needed for PWM, we should change the default TIMER_DEV(0) to `TIM5` above in the timer configuration.